src/ManageCourses.Api/Services/UserInstitution.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using GovUk.Education.ManageCourses.Domain.Models;
+
+namespace GovUk.Education.ManageCourses.Api.Services
+{
+    public class UserInstitution
```

Comment:
This class smells a bit funny but I haven't got any suggestions off the top of my head so :+1: